edenia/eden-spend-explorer
# feat(backend): save income data

### Save income data

### What does this PR do?

- Resolve #194 

### Steps to test

1. run the project
2. See incomes of previous elections when a delegate withdraws money from previous elections.

#### CheckList
